ingest/ingestHelpers.py

```diff
@@ -1,20 +1,15 @@
 __author__ = 'Hao'
```

Comment:
Thank you so much for this advice! That will definitely make it way more organized than now!